Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -109,6 +109,9 @@ class NewTabPageViewController: UIViewController {
   private var preventReloadOnBraveNewsEnabledChange = false
 
   private let notifications: NewTabPageNotifications
+  
+  /// Internal url to access the new tab page.
+  static let ntpInteralURL = URL(string: "\(InternalURL.baseUrl)/\(AboutHomeHandler.path)#panel=0")
```

Comment:
I think this better be inside TabManager as constant instead of static property inside NTPController

Reply:
thanks, was not really sure where to place it